src/doc/src/commands/cargo-locate-project.md

```diff
@@ -10,8 +10,15 @@ cargo-locate-project - Print a JSON representation of a Cargo.toml file's locati
 
 ## DESCRIPTION
 
-This command will print a JSON object to stdout with the full path to the
-`Cargo.toml` manifest.
+This command will print a JSON object to stdout with the full path to the manifest. The
+manifest is found by searching upward for a file named `Cargo.toml` starting from the current
+working directory.
+
+If the project happens to be a part of a workspace, the manifest of the project, rather than
+the workspace root, is output. This can be overriden by the `--workspace` flag. The root
+workspace is found by traversing further upward or by using the field `package.workspace` after
+locating the manifest of a workspace member. In most other commands, cargo locates the project
+root by default, while in `cargo locate-project`, the above flag needst to be specified
```

Comment:
I think it behaves like this.

```suggestion
locating the manifest of a workspace member. In most other commands, cargo locates the workspace
root by default, while in `cargo locate-project`, the above flag needs to be specified.
```

Reply:
Really Sorry. I was wrong by misunderstanding what `cargo locate-project` does 😞. They should behave the same. Could you help me remove things after “In most other commands…”? Thank you :)